core/authority/authority_store_test.go

```
@@ -0,0 +1,576 @@
+		if err != nil {
+			return err
+		}
+		// Quick substring filter before parsing to keep the test
+		// fast on large repos: skip files that don't mention
+		// "AuthorityStore" or ".Put(" on a receiver named store.
+		if !strings.Contains(string(src), "AuthorityStore") {
+			return nil
+		}
+		file, err := parser.ParseFile(fset, path, src, parser.ParseComments)
+		if err != nil {
+			return nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 380) but it returns nil

(nilerr)